src/Util/HTTPSignature.php

```diff
@@ -604,13 +604,14 @@ public static function getKeyIdContact(array $http_headers): array
 	/**
 	 * Gets a signer from a given HTTP request
 	 *
-	 * @param string $content
-	 * @param array $http_headers
+	 * @param string   $content
+	 * @param array    $http_headers
```

Comment:
We should remove phpdocs that adds no value.

Reply:
Documentation added